docs/content/preview/yugabyte-platform/prepare/cloud-permissions/cloud-permissions-nodes-gcp.md

```diff
@@ -82,3 +86,31 @@ If you will be using your own custom SSH keys, then ensure that you have them wh
 | Save for later | To configure |
 | :--- | :--- |
 | Custom SSH keys | [GCP provider configuration](../../../configure-yugabyte-platform/gcp/) |
+
+## GKE service account-based IAM (GCP IAM)
+
+Google Kubernetes Engine (GKE) uses a concept known as "Workload Identity" to provide a secure way to allow a Kubernetes service account (KSA) in your GKE cluster to act as an IAM service account so that your Kubernetes universes can access GCS for backups.
```

Comment:
```suggestion
Google Kubernetes Engine (GKE) uses a concept known as "Workload Identity" to provide a secure way to allow a Kubernetes service account ([KSA](https://kubernetes.io/docs/concepts/security/service-accounts/)) in your GKE cluster to act as an IAM service account so that your Kubernetes universes can access GCS for backups.
```